docs/sources/faq.md

```diff
@@ -259,6 +259,10 @@ If you *really* need to change its setting, make sure to change it
 back to its [default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en) 
 afterwards.
 
+### Why do I get `Connection reset by peer` when making a request to a service running in a container?
+
+Typically, this message is returned if the service is already bound to your localhost. As a result, requests coming to the container from outside are dropped. To correct this problem, change the service's configuration on your localhost so that the service accepts requests from all IPs. 
```

Comment:
@thaJeztah Considered adding the example from this page: [http://ubuntuforums.org/showthread.php?t=2150688]  my only concern was that it was OS specific in some way.   If you say it is ok, I'll add it.

Slapped for the wrap! Darn it not again.  OK, I'll fix that too once you let me know about that example